src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -767,7 +771,7 @@ internal bool TryGetNodeInfo(string name, out CustomNodeInfo info)
                         selectedNodeSet.SelectMany(
                             node =>
                                 Enumerable.Range(0, node.InPorts.Count)
-                                .Where(node.HasConnectedInput)
+                                .Where(index=>node.InPorts[index].Connectors.Any())
```

Comment:
We can't use the `PortModel.IsConnected` method here because that assumes default values as part of connection. Here we _really_ want to know if the port has any connectors, so we use the more idiomatic `Connectors.Any()`